lms/static/sass/base/_mixins.scss

```diff
@@ -135,6 +135,16 @@
   width: 1px;
 }
 
+// extends - hidden el - screenreaders
```

Comment:
**An extend that does this exact job is already in this file** - https://github.com/edx/edx-platform/blob/clrux/UX-1994-multiple-choice-button/lms/static/sass/base/_mixins.scss#L127.

It is not named similarly to our PL extend name, but I think that can be synced up later once the PL is introduced into the LMS. **Please use the existing extend**.